src/Pages/TabsPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link, useParams } from 'react-router-dom';
+import classnames from 'classnames';
+import { Tab } from '../types/Tab';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+const TabsPage = () => {
+  const { tabId } = useParams();
+  let currentTab = tabs.find(tab => tab.id === tabId) as Tab;
```

Comment:
Instead of using `let`, use `const` along with the useState hook to manage the state of the currentTab and trigger re-rendering when the current tab changes.